src/Integration.UnitTests/Integration.UnitTests.csproj

```diff
@@ -26,7 +26,6 @@
     <DefineConstants>DEBUG;TRACE</DefineConstants>
     <ErrorReport>prompt</ErrorReport>
     <WarningLevel>4</WarningLevel>
-    <CodeAnalysisRuleSet>Integration.UnitTests.ruleset</CodeAnalysisRuleSet>
```

Comment:
Can you revert this change please? I prefer to have those than no to have them at all.